docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md

```diff
@@ -0,0 +1,40 @@
+# Sites
+
+## Overview Study
+
+**Sites** or **centres** are terms used interchangeably in LORIS.
+**Sites** are defined in the `psc` table of the database.
+
+LORIS' default schema defines the first psc (CenterID=1) as the
```

Comment:
Maybe this first sentence is the right place to explain what psc stands for?

"LORIS' default schema defines the first project study center (psc)..."

Reply:
The wiki used to start with this line: 
> Study sites or centres are defined in the psc table, with one row per Project Study Centre (PSC).